src/plugins/intel_cpu/src/nodes/fullyconnected.cpp

```diff
@@ -538,6 +538,8 @@ const std::vector<impl_desc_type>& FullyConnected::getDefaultImplPriority() {
         impl_desc_type::gemm_any,
         impl_desc_type::gemm,
         impl_desc_type::jit_gemm,
+        //Lower down brgemm avx2 primitive priority than jit_gemm for perf issue.
+        impl_desc_type::brgemm_avx2,
```

Comment:
Is brgemm_avx2 shows lower performance both for Convolution and FC/MatMul primitives. My understanding was that is mostly applicable for Convolutions. is that correct?

Reply:
It should be. Already prioritize brgemm avx2 higher than jit gemm for FC/Matmal. Internal test shows INT8 precision has some perf&latency GEOMEAN gain. Others precision GEOMEAN doesn't almost  changes. Updated.